backend/audit/models.py

```diff
@@ -82,39 +66,55 @@ class SingleAuditChecklist(models.Model):
     multiple_eins_covered = models.BooleanField(
         verbose_name=_("Multiple EINs covered"), null=True
     )
+    uei = models.CharField(
```

Comment:
Changing to auditee_uei would be ideal, if we can make it work.

Reply:
It means changing up some tests and the API, which I can do. Just references to "UEI" are plentiful, so I want to change the model, not the API endpoint functionality (just the API backend that reads the model)